build/cli.js

```diff
@@ -76,6 +76,17 @@ async function buildDocuments() {
       // which makes this not great and refactor-worthy
       JSON.stringify({ doc: builtDocument })
     );
+    if (document.metadata.contributors) {
```

Comment:
@peterbe This will be a problem once the first content document is created via GitHub (i.e., it has no original Wiki contributors). I'm thinking we should always write a `contributors.txt` document, since we always want to show the GitHub link to the commits right? So, I think we should delete this condition, and modify the `renderContributorsTxt()` function such it can handle the case when the array of Wiki contributors is empty.

Reply:
How about; first remove this `if` statement and then inside `renderContributorsTxt` make it something like this:
```javascript
function renderContributorsTxt(names = null, githubURL = null) {
  console.assert(names || githubURL, "It has to be either")
  ...
}
```

Truth is, there are some documents in either translated-content, archived-content, or mdn/content where there are no contributors. I've seen it crash here before because `document.metadata.contributors` becomes falsy. I suspect that's because some weird docs were only edited by the bot or perhaps the contributions were removed when the user decided to self-destruct. 
I'll figure something out.

Reply:
@peterbe Sounds good to me!